website/docs/guides/1.0.0-upgrade-guide.html.markdown

```diff
@@ -79,6 +79,41 @@ so no changes are needed.
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint)
 **NOTE** Doc links for [mongodbatlas_privatelink_endpoint_service](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service)
 
+## Migration from Cloud Provider Snapshot Backup Policy to Cloud Provider Backup Schedule
+
+To upgrade from v0.9.1 to v1.0.0 with the existing resources `mongodbatlas_cloud_provider_snapshot_backup_policy`
+perform the following steps
+
+1. Remove your existing `mongodbatlas_cloud_provider_snapshot_backup_policy` resources from terraform state
+   **NOTE** before doing any terraform state change please do a backup
+
+```bash
+$ terraform state rm mongodbatlas_cloud_provider_snapshot_backup_policy.test
+```
+
+2. Change the version of provider to the v1.0.0
+
+3. Perform an import from your existing `mongodbatlas_cloud_provider_snapshot_backup_policy` to `mongodbatlas_cloud_backup_schedule`
+```bash
+    terraform import mongodbatlas_cloud_backup_schedule.test <project_id>-<cluster_name>
+```
+
+4. Do a terraform plan (no changes detected)
+   **NOTE** If you imported without policies since now it's optional, it's possible that it will detect changes after terraform plan
```

Comment:
I'm not sure what this line means?  I've tried working though this and it's just not making sense to me.  Perhaps explain the steps and I can rephrase it to be clearer?